Data-Structure/Collection-In-Java/CollectionOfEmployee/SortingBasedOnName.java

```diff
@@ -0,0 +1,24 @@
+package CollectionOfEmployee;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Niraj Gupta
+ * Class to perfrom sorting on employee based on their names.

```

Comment:
It could have been clubbed with other sort cls and named sorting with two methods. Is there any specific reason for creating two classes with one method each?